inst/tinytest/test_deserialization.R

```diff
@@ -23,18 +23,18 @@ simplify_lvl <- list(
 # scalar =======================================================================
 #* integer ---------------------------------------------------------------------
 test <- "1"
-target <- 1L
+target <- 1
```

Comment:
This may matter to us.  Do ints no longer come back as integer?  `1` is a numeric in R.  If you adjust that adjust the preceding comment on line 24 too or move it.